mmselfsup/models/heads/mae_head.py

```diff
@@ -31,7 +26,7 @@ def __init__(self,
         self.loss = MODELS.build(loss)
 
     def patchify(self, imgs: torch.Tensor) -> torch.Tensor:
-        """Get patches of the image."""
+
```

Comment:
Better to add some docstring here, indicate the shape of input tensor and output tensor